Ordbokene dev henrik

henrikaskj opened the pull request.

sawravchy commented on apps/sps-ordbokene-web/components/ArticleHeader.vue:
```diff
@@ -64,6 +56,7 @@ h3 {
 
 .hgno {
```
What about if we use rounded-full with bg-tertiary? Because seems like the current background isn't effective.
<img width="269" alt="issue" src="https://github.com/uib-ub/uib-ub-monorepo/assets/43489196/3722f48e-24de-4f24-be98-3fab72f2c775">
<img width="111" alt="issue" src="https://github.com/uib-ub/uib-ub-monorepo/assets/43489196/c5a2905a-3a58-430c-9f69-874fc063a7e6">

henrikaskj replied:
I'll remove the backround alltogether, it isn't necessary

sawravchy reviewed (commented):
- Dictiontionaries Header Issue:
Maybe it will look much better if we shift this little bit right.
<img width="200" alt="issue" src="https://github.com/uib-ub/uib-ub-monorepo/assets/43489196/7fad0f4e-09cd-49fa-b2e3-84d07438523b">

- 404 page security issue:
We can add a custom 404 page or keep the previous URL redirection method.

henrikaskj commented:
> * Dictionaries Header Issue:
>   Maybe it will look much better if we shift this little bit right.
> 
> <img alt="issue" width="200" src="https://user-images.githubusercontent.com/43489196/262990733-7fad0f4e-09cd-49fa-b2e3-84d07438523b.png">
> * 404 page security issue:
>   We can add a custom 404 page or keep the previous URL redirection method.

I've now tweaked margins and paddings and made some simplifications so that it will be easier to change it in the future, e.g. by using gaps instead of vertical margins/paddings between the articles.

sawravchy commented:
> > * Dictionaries Header Issue:
> >   Maybe it will look much better if we shift this little bit right.
> > 
> > <img alt="issue" width="200" src="https://user-images.githubusercontent.com/43489196/262990733-7fad0f4e-09cd-49fa-b2e3-84d07438523b.png">
> > * 404 page security issue:
> >   We can add a custom 404 page or keep the previous URL redirection method.
> 
> I've now tweaked margins and paddings and made some simplifications so that it will be easier to change it in the future, e.g. by using gaps instead of vertical margins/paddings between the articles. ![image](https://user-images.githubusercontent.com/22298175/263673120-4434941a-bd39-440c-9dbf-88bad00b63eb.png) ![image](https://user-images.githubusercontent.com/22298175/263673345-16ecaaec-0bd0-49b1-8b14-cd5571757e19.png)


Looks much better now but seems like you missed the article hover style.

henrikaskj commented:
> > > * Dictionaries Header Issue:
> > >   Maybe it will look much better if we shift this little bit right.
> > > 
> > > <img alt="issue" width="200" src="https://user-images.githubusercontent.com/43489196/262990733-7fad0f4e-09cd-49fa-b2e3-84d07438523b.png">
> > > * 404 page security issue:
> > >   We can add a custom 404 page or keep the previous URL redirection method.
> > 
> > 
> > I've now tweaked margins and paddings and made some simplifications so that it will be easier to change it in the future, e.g. by using gaps instead of vertical margins/paddings between the articles. ![image](https://user-images.githubusercontent.com/22298175/263673120-4434941a-bd39-440c-9dbf-88bad00b63eb.png) ![image](https://user-images.githubusercontent.com/22298175/263673345-16ecaaec-0bd0-49b1-8b14-cd5571757e19.png)
> 
> Looks much better now but seems like you missed the article hover style.

Yes, I removed it because I think it makes it seem like the article is clickable, which can be a bit confusing. Don't you agree?

henrikaskj commented:
> > > > * Dictionaries Header Issue:
> > > >   Maybe it will look much better if we shift this little bit right.
> > > > 
> > > > <img alt="issue" width="200" src="https://user-images.githubusercontent.com/43489196/262990733-7fad0f4e-09cd-49fa-b2e3-84d07438523b.png">
> > > > * 404 page security issue:
> > > >   We can add a custom 404 page or keep the previous URL redirection method.
> > > 
> > > 
> > > I've now tweaked margins and paddings and made some simplifications so that it will be easier to change it in the future, e.g. by using gaps instead of vertical margins/paddings between the articles. ![image](https://user-images.githubusercontent.com/22298175/263673120-4434941a-bd39-440c-9dbf-88bad00b63eb.png) ![image](https://user-images.githubusercontent.com/22298175/263673345-16ecaaec-0bd0-49b1-8b14-cd5571757e19.png)
> > 
> > 
> > Looks much better now but seems like you missed the article hover style.
> 
> Yes, I removed it because I think it makes it seem like the article is clickable, which can be a bit confusing. Don't you agree?

I think I should add your new shadow style to the list view btw:
![image](https://github.com/uib-ub/uib-ub-monorepo/assets/22298175/e797ad3c-272a-43a1-8b3a-3b05b858912a)

sawravchy commented:
> > Yes, I removed it because I think it makes it seem like the article is clickable, which can be a bit confusing. Don't you agree?
> 
> I think I should add your new shadow style to the list view btw: ![image](https://user-images.githubusercontent.com/22298175/263682468-e797ad3c-272a-43a1-8b3a-3b05b858912a.png)

Yes, I agree with you. I added that shadow because I thought that the article would be clickable, just like our current version.

henrikaskj commented:
> > > Yes, I removed it because I think it makes it seem like the article is clickable, which can be a bit confusing. Don't you agree?
> > 
> > 
> > I think I should add your new shadow style to the list view btw: ![image](https://user-images.githubusercontent.com/22298175/263682468-e797ad3c-272a-43a1-8b3a-3b05b858912a.png)
> 
> Yes, I agree with you. I added that shadow because I thought that the article would be clickable, just like our current version.

I see. In the current version it's only the heading that is clickable. I removed it for two reasons:
1. Cases like this, where there are two headings, wich can make users think there are two links instead of one:
![image](https://github.com/uib-ub/uib-ub-monorepo/assets/22298175/26f63832-44bc-4de8-a9cb-830cb75a7bc4)
2. Clicking the link doesn't give you any more information than what you see in the result list.

If we don't bring back the links we could instead add a button in the article footer saying "show separately"

sawravchy reviewed (commented):
It looks much better than the previous version, except for this button text issue. However, I'm not sure whether it will cause an accessibility issue or not. For me, it seems to be okay. 

During testing, my screen (1920 x 1200) was zoomed in at 110-125%.


<img width="869" alt="issue" src="https://github.com/uib-ub/uib-ub-monorepo/assets/43489196/dbb1134d-30ff-4c9e-ada5-46a4607c6477">
